feature/tabsflow/moreflow/faq/src/main/java/com/epmedu/animeal/tabs/more/faq/presentation/FAQScreenUI.kt

```diff
@@ -0,0 +1,84 @@
+package com.epmedu.animeal.tabs.more.faq.presentation
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.statusBarsPadding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.foundation.layout.LastElementBottom
+import com.epmedu.animeal.foundation.preview.AnimealPreview
+import com.epmedu.animeal.foundation.theme.AnimealTheme
+import com.epmedu.animeal.foundation.util.generateLoremIpsum
+import com.epmedu.animeal.resources.R
+import com.epmedu.animeal.tabs.more.faq.domain.model.FrequentlyAskedQuestion
+import com.epmedu.animeal.tabs.more.faq.presentation.ui.FAQHeader
+import com.epmedu.animeal.tabs.more.faq.presentation.ui.FAQListItem
+import com.epmedu.animeal.tabs.more.faq.presentation.viewmodel.FAQState
+import kotlinx.collections.immutable.toImmutableList
+
+@Composable
+internal fun FAQScreenUI(
+    state: FAQState,
+    onBack: () -> Unit
+) {
+    var selectedQuestion: FrequentlyAskedQuestion? by remember { mutableStateOf(null) }
```

Comment:
Does it survive screen orientation changes, may be use `remeberSaveable`?

Reply:
`rememberSaveable` crashes the app, so I extracted `selectedQuestion` to the `FAQState`